test/sqllogictest/privilege_checks.slt

```diff
@@ -1812,12 +1830,18 @@ COMPLETE 0
 simple conn=joe,user=joe
 EXPLAIN SELECT * FROM v;
 ----
-db error: ERROR: permission denied for TABLE "materialize.public.t"
+Explained Query:
+  ReadStorage materialize.public.t
+
+COMPLETE 1
 
 simple conn=child,user=child
 EXPLAIN SELECT * FROM v;
 ----
-db error: ERROR: permission denied for TABLE "materialize.public.t"
+Explained Query:
+  ReadStorage materialize.public.t
+
+COMPLETE 1
 
 simple conn=mz_system,user=mz_system
```

Comment:
You can remove this grant and all revokes, except `REVOKE CREATE ON SCHEMA materialize.public FROM view_owner;`